packages/quantic/force-app/examples/main/lwc/exampleQuanticCaseClassification/exampleQuanticCaseClassification.html

```diff
@@ -1,16 +1,45 @@
 <template>
-  <c-example-layout title={pageTitle} description={pageDescription} show-preview={isConfigured}>
-    <c-configurator slot="configuration" options={options} ontryitnow={handleTryItNow}>
-      <c-action-fetch-classifications slot="actions" disabled={notConfigured} engine-id={config.engineId}>
-      </c-action-fetch-classifications>
+  <c-example-layout
+    title={pageTitle}
+    description={pageDescription}
+    show-preview={isConfigured}
+  >
+    <c-configurator
+      slot="configuration"
+      options={options}
+      ontryitnow={handleTryItNow}
+    >
+      <c-action-fetch-classifications
+        slot="actions"
+        disabled={notConfigured}
+        engine-id={config.engineId}
+      ></c-action-fetch-classifications>
+      <c-action-fetch-suggestions
+        slot="actions"
+        disabled={notConfigured}
+        engine-id={config.engineId}
+      ></c-action-fetch-suggestions>
```

Comment:
Same thing here, don't think this is necessary. 
In the test we only want to confirm that a new request for document suggestion has been made when the value of classification changes, we do not need to mock this document suggestion request. we only need to to confirm that the request has been made.